spec.md

```diff
@@ -0,0 +1,52 @@
+# Open Container Initiative Image Format Specification
+
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
+
+The goal of this specification is to enable the creation of interoperable tools for building, transporting, and preparing a container image to run.
+
+## Table of Contents
+
+- [Introduction](spec.md)
+- [Notational Conventions](#notational-conventions)
+- [Overview](#overview)
+	- [Understanding the Specification](#understanding-the-specification)
+    - [Media Types](media-types.md)
```

Comment:
Inconsistent indent?  You should replace the earlier tab with four spaces (opencontainers/runtime-spec#495).